modules/cluster-logging-rn-5.5.6.adoc

```diff
@@ -0,0 +1,53 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-6_{context}"]
+= Logging 5.5.6
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0386[OpenShift Logging Bug Fix Release 5.5.6].
+
+== Known issues
+[id="openshift-logging-5-5-6-known-issues"]
+
+
+[id="openshift-logging-5-5-6-bug-fixes"]
+== Bug fixes
+* Before this update, the Pod Security admission controller added the label `podSecurityLabelSync = true` to the `openshift-logging` namespace. This resulted in our specified security labels being overwritten, and as a result Collector pods would not start. With this update, the label `podSecurityLabelSync = false` preserves security labels. Collector pods deploy as expected. (link:https://issues.redhat.com/browse/LOG-3340[LOG-3340])
+
+* Before this update, the Operator installed the console view plugin, even when it was not enabled on the cluster. This caused the Operator to crash. With this update, if an account for a cluster does not have the console view enabled, the Operator functions normally and does not install the console view. (link:https://issues.redhat.com/browse/LOG-3407[LOG-3407])
+
+* Before this update, a prior fix to support a regression where the status of the Elasticsearch deployment was not being updated caused the Operator to crash unless the `Red Hat Elasticsearch Operator` was deployed. With this update, that fix has been reverted so the Operator is now stable but re-introduces the previous issue related to the reported status. (link:https://issues.redhat.com/browse/LOG-3428[LOG-3428])
+
+* Before this update, when the Loki Operator deployed with a size of `1x.small` only one pod deployed for the LokiStack gateway. With this update, two pods deploy for the gateway. (link:https://issues.redhat.com/browse/LOG-3478[LOG-3478])
```

Comment:
This fix actually applies to all the stack sizes. The bug was that only one replica of the Gateway was created regardless of the size. The fix makes it so that the replica count of the Gateway is taken from the size's actual count.

Reply:
```suggestion
* Before this update, the Loki Operator only deployed one replica of the LokiStack gateway. With this update, the number of replicas is correctly configured according to the selected size. (link:https://issues.redhat.com/browse/LOG-3478[LOG-3478])
```

Reply:
```suggestion
* Before this update, the Loki Operator only deployed one replica of the LokiStack gateway regardless of the chosen stack size. With this update, the number of replicas is correctly configured according to the selected size. (link:https://issues.redhat.com/browse/LOG-3478[LOG-3478])
```

Reply:
I just took your suggestion @xperimental and added a bit at the end of the second sentence.

Reply:
Thanks team!